xgcm/test/test_faceconnections.py

```diff
@@ -211,19 +229,59 @@ def test_diff_interp_connected_grid_x_to_y(ds, ds_face_connections_x_to_y):
     # TODO: checking all the other boundaries
 
 
-@pytest.mark.xfail(reason="connected grids not implemented for grid ufunc refactor yet")
+# TODO: Relaease the periodic test here once we unified the API with padding.
+@pytest.mark.parametrize(
+    "boundary", [pytest.param("periodic", marks=pytest.mark.xfail), "fill"]
+)
```

Comment:
I dont' really understand why this is currently xfailed

Reply:
I did not either. This is actually one of the issues that I think I need to resolve here. Lets chat about this today?

Reply:
I was able to unxfail this with my latest changes